client/src/components/Overview/StyleSelector.jsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-function StyleSelector() {
-  return null;
+function Style({ style, handleStyleSelect, selected }) {
+  // TODO: handle selected
+  return (
+    <div onClick={() => handleStyleSelect(style.style_id)}>
+      <img
+        src={style.photos[0].thumbnail_url}
+        alt={`${style.name} style thumbnail`}
+        width="50"
+      />
+    </div>
+  );
 }
 
+Style.propTypes = {
+  style: PropTypes.object.isRequired,
+  selected: PropTypes.bool.isRequired,
+  handleStyleSelect: PropTypes.func.isRequired,
+};
+
+function StyleSelector({ styles, selectedStyleId, handleStyleSelect }) {
+  // TODO: use color-thief to extract average color of thumbnails server-side
+  const stylesArray = styles.map((style) => (
+    <Style
+      key={style.style_id}
+      style={style}
+      handleStyleSelect={handleStyleSelect}
+      selected={style.styled_id === selectedStyleId}
+    />
+  ));
+
+  return (
+    <div>
+      { stylesArray }
```

Comment:
Put this directly in here for easier readability. But seems this component is maybe not even needed as it's essentially a map function over the `<Style>` component. Is this `<StyleSelector>` going into a lot of places?

Reply:
No, StyleSelector is only used once. I did it like this to try to compartmentalize things and keep `Overview` simple by just including `StyleSelector`. Would you recommend replacing `<StyleSelector />` by the map in `Overview`?

Reply:
Actually, I realize now that's not even what I'm doing. Taking your other comment into account, I'll move `<StyleSelector />` to `<Overview />`.